src/main/resources/log4j.properties

```diff
@@ -0,0 +1,12 @@
+# Уровень логирования
+log4j.rootLogger=INFO, file
+
+# Апендер для работы с файлами
+log4j.appender.file=org.apache.log4j.RollingFileAppender
+# Путь где будет создаваться лог файл
+log4j.appender.file.File=C:\\TMP\\log_file.log
+# Указываем максимальный размер файла с логами
+log4j.appender.file.MaxFileSize=1MB
+# Конфигурируем шаблон вывода логов в файл
+log4j.appender.file.layout=org.apache.log4j.PatternLayout
+log4j.appender.file.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
```

Comment:
Add a blank line at the end of the file and please remove all Russian explanation or removed it by an explanation on English.

Reply:
why we need to add blank lines?
fixed